x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/SpatialPlugin.java

```diff
@@ -194,7 +196,11 @@ public Map<String, Processor.Factory> getProcessors(Processor.Parameters paramet
 
     @Override
     public List<GenericNamedWriteableSpec> getGenericNamedWriteables() {
-        return List.of(new GenericNamedWriteableSpec(CartesianBoundingBox.class.getSimpleName(), CartesianBoundingBox::new));
+        return List.of(
+            new GenericNamedWriteableSpec(CartesianBoundingBox.class.getSimpleName(), CartesianBoundingBox::new),
```

Comment:
Ah sorry I missed this in the previous PR. We need to hard-code the name as a string rather than generating it dynamically from the class like this. Otherwise we might rename the class in the future which would break the transport protocol.

Reply:
Ah, good point!

Reply:
Created a simple PR for this at https://github.com/elastic/elasticsearch/pull/99408